cluster/gce/gci/configure-helper.sh

```diff
@@ -1160,6 +1160,7 @@ function start-kubelet {
   local -r kubelet_env_file="/etc/default/kubelet"
   local kubelet_opts="${KUBELET_ARGS} ${KUBELET_CONFIG_FILE_ARG:-}"
   echo "KUBELET_OPTS=\"${kubelet_opts}\"" > "${kubelet_env_file}"
+  echo "KUBE_COVERAGE_FILE=\"/var/log/kubelet.cov\"" >> "${kubelet_env_file}"
```

Comment:
It looks like this is going to enable coverage unconditionally. Is this something that should be enabled in production clusters or just in testing clusters?

@dchen1107 @abgworrall

Reply:
It won't; coverage needs to be enabled at compile-time. This only affects where coverage will be dumped to if coverage is already enabled.